site/docs/4.9.0/overview/releaseNotes.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Apache BookKeeper 4.9.0 Release Notes
+---
+
+This is the 16th release of Apache BookKeeper!
+
+The 4.9.0 release incorporates hundreds of bug fixes, improvements, and features since previous major release, 4.8.0,
+which was released four months ago. It is a new milestone in Apache BookKeeper community. This release includes great
+new features, like 
+
+Apache BookKeeper/DistributedLog users are encouraged to [upgrade to 4.9.0](../../admin/upgrade). The technical details of
+this release are summarized below.
+
+## Highlights
+
+The main features in 4.9.0 cover are around following areas:
+
+- Dependencies Changes
+- Public API
+- Configuration
+- Metadata
+- Table Service
+- Operations
+- Builds & Testing
+- Enhancements
+- Bug Fixes
+
+### Dependencies Changes
+
+Here is a list of dependencies changed in 4.9.0:
+
+- Upgrade [Jackson](http://fasterxml.com/) from `2.8.9` to `2.9.7`.
+- Upgrade [Jline](https://jline.github.io/) to `2.11`.
+- Upgrade [Netty](https://netty.io/) from `4.1.22` to `4.1.31`.
+- Upgrade [TestContainers](https://www.testcontainers.org/) from `1.7.0` to `1.8.3`.
+
+### Public API
+
+There are multiple new client features introduced in 4.9.0. Here are two highlighted features:
+
+- LedgerHandleAdv exposes `asyncAddEntry` variant that takes ByteBuf
+
+### Configuration
+
+There are bunch of new settings introduced in both bookie and client in 4.9.0. Here are those settings:
+
+#### Bookie
+
+- `serverNumIOThreads`: configures the number of IO threads for bookies
+  (see [#1612](https://github.com/apache/bookkeeper/pull/1612))
+- The default value of `fileInfoFormatVersionToWrite` is bumped from `0` to `1`.
+  (see [#1689](https://github.com/apache/bookkeeper/pull/1689))
+- The default value of `journalFormatVersionToWrite` is bumped from `5` to `6`.
+  (see [#1689](https://github.com/apache/bookkeeper/pull/1689))
+
+#### Client
```

Comment:
Didn't we also introduce the common allocator stuff from @merlimat ?

Reply:
@eolivelli will add